### PR TITLE
media-gfx/opentoonz: version bump to 1.3.0

### PR DESCRIPTION
**Pull request by otakuto**
Closes: https://bugs.gentoo.org/677030
Package-Manager: Portage-2.3.52, Repoman-2.3.12

### CODE REVIEW

File: media-gfx/opentoonz/opentoonz-1.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	media-libs/libpng:=
+	media-libs/libsdl2:=
+	sci-libs/cblas-reference:=
+	>=sci-libs/superlu-4.1:=
+	sys-libs/zlib:=
+	virtual/libusb:=
+	virtual/opengl
+"
+DEPEND="
+	$RDEPEND
```
[the remark this answers is not in the document]
Comment (otakuto) [reply media-gfx/opentoonz/opentoonz-1.3.0.ebuild]: I fixed it.

File: media-gfx/opentoonz/opentoonz-1.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+HOMEPAGE="https://github.com/opentoonz/opentoonz"
+SRC_URI="https://github.com/opentoonz/opentoonz/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD libtiff"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	app-arch/lz4
```
Comment (a17r) [inline media-gfx/opentoonz/opentoonz-1.3.0.ebuild]: missing slot operator, why did you drop it?

File: media-gfx/opentoonz/opentoonz-1.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+	>=dev-libs/boost-1.55.0
```
Comment (a17r) [inline media-gfx/opentoonz/opentoonz-1.3.0.ebuild]: missing slot operator, why did you drop it?

File: media-gfx/opentoonz/opentoonz-1.3.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+	>=dev-qt/qtnetwork-5.9:5
+	>=dev-qt/qtopengl-5.9:5
+	>=dev-qt/qtprintsupport-5.9:5
+	>=dev-qt/qtscript-5.9:5
+	>=dev-qt/qtsvg-5.9:5
+	>=dev-qt/qtwidgets-5.9:5
+	>=dev-qt/qtxml-5.9:5
+	media-libs/freeglut
+	media-libs/freetype:2
+	media-libs/glew
```
Comment (a17r) [inline media-gfx/opentoonz/opentoonz-1.3.0.ebuild]: missing slot operator, why did you drop it?

File: media-gfx/opentoonz/opentoonz-1.3.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+	media-libs/libjpeg-turbo
+	>=media-libs/libmypaint-1.3.0
+	media-libs/libpng
```
Comment (a17r) [inline media-gfx/opentoonz/opentoonz-1.3.0.ebuild]: missing slot operator, why did you drop it?

File: media-gfx/opentoonz/opentoonz-1.3.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+	media-libs/libsdl2
+	sci-libs/cblas-reference
+	>=sci-libs/superlu-4.1
```
Comment (a17r) [inline media-gfx/opentoonz/opentoonz-1.3.0.ebuild]: missing slot operator, why did you drop it?

**Comment by a17r** [comment]
ping

**Comment by otakuto** [comment]
Just a moment, please.😣

**Comment by otakuto** [comment]
Thank you for waiting.
I misunderstood that multiple lines.

**Comment by a17r** [comment]
thanks!

**Pull request closed without merging**